### PR TITLE
styled Component in MobileTab.react.js

### LINKED ISSUE #2641: styled Component in MobileTab.react.js
**Actual Behaviour**

styled Component is not created in MobileTab.react.js

**Expected Behaviour**

styled Component in MobileTab.react.js

**Would you like to work on the issue?**

Yes

### PR DESCRIPTION
**Pull request by samagragupta**
Fixes #2641

Changes: Created styled Component in MobileTab.react.js

Demo Link: https://pr-2713-fossasia-susi-web-chat.surge.sh

### CODE REVIEW

File: src/components/Settings/MobileTab.react.js [hunk 1]
```diff
@@ -20,6 +20,63 @@ import MenuItem from '@material-ui/core/MenuItem';
...
+`;
+
+const CountryCode = styled.div`
+  margin-top: 2rem;
+  font-size: 14px;
+  display: flex;
+  align-items: center;
+`;
+
+const CountryText = styled.div`
```
Comment (fragm3) [inline src/components/Settings/MobileTab.react.js]: Undo all 1 line style changes

File: src/components/Settings/MobileTab.react.js [hunk 2]
```diff
@@ -20,6 +20,63 @@ import MenuItem from '@material-ui/core/MenuItem';
 import { sortCountryLexographical } from '../../utils/helperFunctions';
 import { isPhoneNumber } from '../../utils';
 import { setUserSettings } from '../../apis';
+import styled from 'styled-components';
+
+const MobileDes = styled.div`
```
Comment (fragm3) [inline src/components/Settings/MobileTab.react.js]: MobileDes? name it correctly

File: src/components/Settings/MobileTab.react.js [hunk 3]
```diff
@@ -20,6 +20,47 @@ import MenuItem from '@material-ui/core/MenuItem';
 import { sortCountryLexographical } from '../../utils/helperFunctions';
 import { isPhoneNumber } from '../../utils';
 import { setUserSettings } from '../../apis';
+import styled from 'styled-components';
+
+const MobileDescription = styled.div`
+  margin-top: 0px;
```
Comment (fragm3) [inline src/components/Settings/MobileTab.react.js]: Not required, remove margin changes and make it inline style

Comment (samagragupta) [reply src/components/Settings/MobileTab.react.js]: @fragm3 Made the changes.

**Pull request merged**